test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesWithBasePathTest.cs

```diff
@@ -420,14 +419,16 @@ public async Task PageConventions_CustomizedModelCanPostToHandlers()
             var token = AntiforgeryTestHelper.RetrieveAntiforgeryToken(await getPage.Content.ReadAsStringAsync(), "");
             var cookie = AntiforgeryTestHelper.RetrieveAntiforgeryCookie(getPage);
 
-            var message = new HttpRequestMessage(HttpMethod.Post, "/CustomModelTypeModel");
-            message.Content = new FormUrlEncodedContent(new Dictionary<string, string>
+            var message = new HttpRequestMessage(HttpMethod.Post, "/CustomModelTypeModel")
```

Comment:
The other tests using `/CustomModelTypeModel` aren't affected by the validation failure.